docs/interchange.rst

```diff
@@ -108,16 +120,16 @@ more detail.
 Table definitions
 =================
 
-
 .. _sec_node_table_definition:
 
 Node Table
 ----------
 
-A **node** defines a specific ancestor that was born at some time in
+A **node** defines single chromosome in a specific ancestor
+that was born at some time in
```

Comment:
A node might be a set of non-homologous chromosomes; but it should be the set that are inherited together. So, maybe "A **node** defines the haploid set of chromosomes of a specific individual that was born at some time in the past: the set of chromosomes inherited from a particular one of the individual's parents."